taskcluster/ci/evaluate-teacher-ensemble/kind.yml

```diff
@@ -110,13 +109,11 @@ tasks:
             unique-kinds: false
             kinds:
                 - finetune-teacher
-                - train-vocab
             fetches:
                 finetune-teacher:
                     - artifact: final.model.npz.best-{best_model}.npz
                       dest: model{this_chunk}
                     - artifact: final.model.npz.best-chrf.npz.decoder.yml
-                train-vocab:
```

Comment:
praise: Thank you for cleaning this up here, and elsewhere. It's nice to avoid the unnecessary direct dependency on `train-vocab`.